mesonbuild/environment.py

```diff
@@ -12,14 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os, re, subprocess, platform
-from . import coredata
-from . import mesonlib
-from . import mlog
-from .compilers import *
-from .mesonlib import EnvironmentException, Popen_safe
 import configparser
+import os
+import platform
+import re
 import shutil
+import subprocess
+
+from . import coredata, mesonlib, mlog
+from .compilers import *
```

Comment:
__[flake8]__
* [F403] 'from compilers import *' used; unable to detect undefined names

<!-- Comment by [SideCI](https://sideci.com) -->